agave/fastapi/rest_api.py

```
@@ -28,6 +31,10 @@
 
 
 class RestApiBlueprint(APIRouter):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(route_class=LoggingRoute, *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)